safemysql.class.php

```diff
@@ -834,7 +834,7 @@ private function createMultiRow($data)
 			if (!is_array($row))
 			{
 				$this->error("Elements of array passed to MultiRow (?m) placeholder should be arrays; ".
-				             gettype($row)." given");
+										 gettype($row)." given");
```

Comment:
No - I hadn't realised that the existing code was using the whole tabs-for-indentation-and-spaces-for-alignment paradigm. Ugh. Will fix.